frontend/components/ExpandableArea.jsx

```diff
@@ -24,18 +24,18 @@ class ExpandableArea extends React.Component {
 
   render() {
     return (
-      <div className="expandable-area">
+      <div className="usa-accordion">
```

Comment:
Minor styling regression with the Advanced Settings section having too much left padding/margin

<img width="886" alt="screen shot 2018-01-22 at 5 43 22 pm" src="https://user-images.githubusercontent.com/697848/35250644-441456de-ff9d-11e7-9865-b99308578d77.png">